dist/mapbox-gl.css

```diff
@@ -5,6 +5,12 @@
     -webkit-tap-highlight-color: rgba(0,0,0,0);
 }
 
+.mapboxgl-canvas-container {
+    -webkit-transform-style: preserve-3d;
+    -moz-transform-style: preserve-3d;
+    transform-style: preserve-3d;
```

Comment:
I'm glad to hear that performance of the `z-index` option is good! 🎉 

We do not feel comfortable including both the `z-index` and `transform-style` options. Having two options will roughly double the amount of testing, support, and debugging required by Mapbox & our contributors. Let's drop the option and stick with just `z-index` for now.